src/pages/LandingPages/MGCovid19/MGCovid19.vue

```diff
@@ -0,0 +1,285 @@
+<template>
+	<www-page id="co19-landing">
+		<kv-hero class="mg-hero" :class="{'experiment':isExperimentActive}">
+			<!-- TODO Update to handle new images and design -->
+			<template v-slot:images>
+				<kv-responsive-image
+					:images="heroImages"
+					alt="A woman in a yellow dress with a look of pride and satisfaction on her face "
+				/>
+			</template>
+			<template v-slot:overlayContent>
+				<div class="row">
+					<div class="overlay-column columns medium-10 medium-offset-1 large-6 large-offset-0 xlarge-5">
+						<p class="mg-headline" v-html="pageCopy.headline">
+						</p>
+						<p class="mg-subhead">
+							{{ pageCopy.subhead }}
+						</p>
+						<landing-form
+							:amount.sync="monthlyGoodAmount"
+							:selected-group.sync="selectedGroup"
```

Comment:
The selected group prop isn't used and can be removed. Key can also be removed since there is now only 1 form on the page.